tests/unit/testsuite/Bolt/Boltpay/ProductProvider.php

```diff
@@ -43,19 +43,9 @@ public static function createDummyProduct($sku, $additionalData = array())
                 self::$websiteIds[] = $website->getId();
             }
         }
-
```

Comment:
All changes in this ProductProvider class are unrelated to adding a new discount.  Please remove this from this PR and place it in a separate PR.